Assets/Scripts/TroopCounter.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using UnityEngine;
+
+
+public class TroopCounter : MonoBehaviour
+{
+    [HideInInspector] public int FirstArmyCount;
```

Comment:
it should not be a field, since you do not serialize it
make it a {get; private set;} property